nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -162,12 +162,14 @@ let
   isolinuxCfg = concatStringsSep "\n"
     ([ baseIsolinuxCfg ] ++ optional config.boot.loader.grub.memtest86.enable isolinuxMemtest86Entry);
 
+  refindBinary = if targetArch == "x64" || targetArch == "aa64" then "refind_${targetArch}.efi" else null;
```

Comment:
Maybe we should just throw here in case targetArch is something else. null will surely not work, right?

Reply:
It will work, we're simply not adding rEFInd if build for e.g. armv7l. That's already what it did, but worse, previously. It wouldn't add the rEFInd binary for AArch64, but would keep the x86 menu entry!